docs/modeling/connectors/ootb/content.md

```diff
@@ -0,0 +1,367 @@
+| `primaryHierarchy` | An array of parent nodes | Array | ioos2a1-2a22-4f42-820d-c2b628751121, 164e0082-5ac1-4afa-a09b-619a8c79f66f | 
+| `properties` | The properties of the created node. | JSON | |
+| `affectedPropertiesBefore` | The properties of the node before the event. | JSON | | 
+| `affectedPropertiesAfter` | The properties of the node after the event. | JSON | | 
+| `aspects` | The aspects of the created node. | Array | | 
+| `aspectNamesBefore` | A list of the aspect names before the event. | Array | versionable | 
+| `aspectNamesAfter` | A list of the aspect names after the event. | Array | versionable, reviewed | 
+
+### Content updated
+This event is fired when the properties of a file or folder in Alfresco Content Services are updated to match the input parameters defined in the trigger event.
+
+For a file the event is called `FILE_UPDATED`. 
+
+For a folder the event is called `FOLDER_UPDATED`.
+
+#### Input parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `properties` | The properties that must be matched for the event to be fired. | JSON | | 
+| `parentFolder` | The node ID of the parent folder. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
+| `nodeType` | The content type of the node that is updated. | String | cm:file | 
+
+#### Output parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `type` | The Java class of the resource. | String | |
+| `id` | The node ID of the node that is updated. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
+| `nodeType` | The content type that is updated. | String | cm:file | 
+| `isFile` | A flag for whether the node is a file. | Boolean | True | 
+| `isFolder` | A flag for whether the node is a folder. | Boolean | False | 
+| `primaryHierarchy` | An array of parent nodes | Array | ioos2a1-2a22-4f42-820d-c2b628751121, 164e0082-5ac1-4afa-a09b-619a8c79f66f | 
+| `properties` | The properties of the created node. | JSON | |
+| `affectedPropertiesBefore` | The properties of the node before the event. | JSON | | 
+| `affectedPropertiesAfter` | The properties of the node after the event. | JSON | | 
+| `aspects` | The aspects of the updated node. | Array | | 
+| `aspectNamesBefore` | A list of the aspect names before the event. | Array | versionable | 
+| `aspectNamesAfter` | A list of the aspect names after the event. | Array | versionable, reviewed | 
+
+### Contet moved
+This event is fired when a file or folder is moved to a new location in Alfresco Content Services that matches the input parameters defined in the trigger event.
+
+For a file the event is called `FILE_MOVED`. 
+
+For a folder the event is called `FOLDER_MOVED`.
+
+#### Input parameters
+
+| Parameter | Description | Type | Example |
+| --------  | ----------- | ---- | ------- |
+| `sourceFolder` | The node ID of the folder the node originates from. | String | dioos2a1-2a22-4f42-820d-c2b628751121 | 
```

Comment:
At the moment, it is not possible to filter with source folder.

Reply:
I'm not sure I understand this one - do you not need the source to fire the event?

Reply:
At the moment, this information is not in the ACS event. So we cannot manage it. In the future, we will have it.

Reply:
Ah ok. I've removed that parameter then until we support it, thanks!